guides/common/assembly_configuring-and-setting-up-remote-jobs.adoc

```diff
@@ -34,8 +34,10 @@ Use one of the following methods to distribute the public SSH key from {SmartPro
 . xref:distributing-ssh-keys-for-remote-execution-manually_{context}[].
 . xref:using-the-api-to-obtain-ssh-keys-for-remote-execution_{context}[].
 . xref:configuring-a-kickstart-template-to-distribute-ssh-keys-during-provisioning_{context}[].
+ifdef::satellite[]
```

Comment:
Can you explain this to me?

Reply:
The link does not resolve in [Foreman docs](https://docs.theforeman.org/nightly/Managing_Hosts/index-foreman-el.html#ssh-keys-for-remote-execution-overview_managing-hosts) but works for [Satellite](https://access.redhat.com/documentation/en-us/red_hat_satellite/6.9/html/managing_hosts/configuring-and-setting-up-remote-jobs_managing-hosts#ssh-keys-for-remote-execution-overview_managing-hosts). I did not find where the reference should point to when running `git grep -i -n "registering-a-host-to-project-using-the-global-registration-template"`. Maybe this section is not included upstream (yet)?

Reply:
Hmm let me investigate.

Reply:
OK so it was introduced with this PR: https://github.com/theforeman/foreman-documentation/pull/359 
I'm assuming that this was changed since the release of 2.3 - let me try and get the full history. This won't work for Satellite 6.10 and if it's in master, we need to fix it.

Reply:
OK, because of changes here:  https://github.com/theforeman/foreman-documentation/pull/514/files#diff-2a16b75ea9053ac4e390eb13cb96398fb7d83baa05c28470b4458b88ac2a5b55L1

we need to update the link to: 
registering-a-host_{context}